File: src/main/java/main/audiohandlers/sources/resume/ResumeTrack.java
package main.audiohandlers.sources.resume;

import com.github.topisenpai.lavasrc.mirror.MirroringAudioSourceManager;
import com.github.topisenpai.lavasrc.mirror.MirroringAudioTrack;
import com.sedmelluq.discord.lavaplayer.track.AudioTrackInfo;

public class ResumeTrack extends MirroringAudioTrack {

    public ResumeTrack(AudioTrackInfo trackInfo, String isrc, String artworkURL, MirroringAudioSourceManager sourceManager) {
        super(trackInfo, isrc, artworkURL, sourceManager);
    }
}

File: src/main/kotlin/main/audiohandlers/sources/resume/ResumeTrack.kt
package main.audiohandlers.sources.resume

import com.github.topisenpai.lavasrc.mirror.MirroringAudioSourceManager
import com.github.topisenpai.lavasrc.mirror.MirroringAudioTrack
import com.sedmelluq.discord.lavaplayer.track.AudioTrackInfo

data class ResumeTrack(
    val audioTrackInfo: AudioTrackInfo,
    val isrc: String?,
    val artwork: String?,
    val sourceManager: MirroringAudioSourceManager
) : MirroringAudioTrack(audioTrackInfo, isrc, artwork, sourceManager)
